tests/jerry/es2015/regression-test-issue.3356.js

```diff
@@ -0,0 +1,19 @@
+// Copyright JS Foundation and other contributors, http://js.foundation
```

Comment:
The file name is incorrect. It should be `regression-test-issue-3356.js` , notice the dash before the issue number.